### PR TITLE
Regular updates and bug fix

### PR DESCRIPTION
**Pull request by YvonneWanggg**
Updated the minimum wage, parameters. Fixed a small bug

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -1,385 +1,392 @@
...
+    
+    if (input$WFFBEN_SQ != "WFF") {
+      SQ_params_with_no_IWTC <- remove_IWTC_from_params(parameters$parameters_SQ)
+      X_SQ_without_IWTC <- hot_emtr(SQ_params_with_no_IWTC)
+      
+      if (input$WFFBEN_SQ == "Max") {
+        # Choose which of benefit or IWTC gives max net income
+        X_SQ <- choose_IWTC_or_benefit(X_SQ, X_SQ_without_IWTC)
+        X_SQ[, EMTR := 1 - 1L*(shift(Net_Income,1L,type="lead")-Net_Income)]
+        X_SQ[, EMTR := zoo::na.locf(EMTR)]
```
Comment (Luke-Symes-Tsy) [inline server.R:139]: Suggest you move this into the "choose_IWTC_or_benefit" function, same with line 153-154

**Review by Luke-Symes-Tsy — APPROVED** [review]
Ok I have spot checked parameter files for TY24-27, looks ok to me. Good work!

**Comment by MeghanStephensNZ** [comment]
The default minimum wage is still potentially misleading - it needs to be changes in ui.R (line 4). Ideally it would reset based on the tax year, but an easier fix would be to make sure its at the current minimum wage ($22.70)

File: server.R (hunk 1 above)
Comment (MeghanStephensNZ) [reply server.R:139]: I agree with Luke that you could move this into the function.

**Review by MeghanStephensNZ — CHANGES_REQUESTED** [review]
I can approve if you update the default minimum wage in the gui.

File: server.R (hunk 1 above)
Comment (YvonneWanggg) [reply server.R:139]: Will update this

**Review by MeghanStephensNZ — APPROVED** [review]

**Comment by MeghanStephensNZ** [comment]
@YvonneWanggg Looks good :D

**Pull request merged**